### ISSUE TITLE
adding nvr

### ISSUE THREAD

**Issue by mrpp** [maciej-or/hikvision_next#245]
I have two nvrs one with 6 cameras one with 4, both nvrs on fixed ip(different ips) I can add my main nvr with 6 cameras but trying to add the second i get this. 
![Settings-–-Home-Assistant (6)](https://github.com/user-attachments/assets/db8254b0-5fc7-4877-b49c-7b628c314486)

**Comment by maciej-or** [comment]
Have you added individual cameras? If they are connected via NVR, you only need to add the NVRs

**Comment by mrpp** [comment]
Yes via nvr, one it adds, but the other on a different ip it won't add. The nvr that is added only 3 of the 6 cameras sensors seem to work. Sent from my HONOR phone-------- Original message --------From: Maciej ***@***.***>Date: Thu, 14 Nov 2024, 20:19To: maciej-or/hikvision_next ***@***.***>Cc: mrpp ***@***.***>, Author ***@***.***>Subject: Re: [maciej-or/hikvision_next] adding nvr (Issue #245)
Have you added individual cameras? If they are connected via NVR, you only need to add the NVRs
